python/llm/example/transformers/transformers_int4_pipeline_readme.md

```diff
@@ -0,0 +1,38 @@
+# INT4 Inference Pipeline for Large Language Model using BigDL-LLM Transformers Style API
+
+In this example, we show a pipeline to convert a large language model to low precision (INT4), and then conduct inference on the converted INT4 model, using BigDL-LLM transformers style API.
```

Comment:
In this example, we show a pipeline to apply BigDL-LLM INT4 optimizations to any Hugging Face Transformers model, and then run inference on the optimized INT4 model.

Reply:
> In this example, we show a pipeline to apply BigDL-LLM INT4 optimizations to any Hugging Face Transformers model, and then run inference on the optimized INT4 model.

Done.

Reply:
Not updated?

Reply:
> Not updated?

Sorry for mistake in updating the file before, now it has been updated.